### ISSUE TITLE
Display @ - value (Axis raw value) on input assignments

### ISSUE THREAD

**Issue by joeherwig** [MobiFlight/MobiFlight-Connector#1509]
Hi Stefan,

### Story

As a user assigning analogue axis values to RPN scripts, it would be great to have the current axis value displayed somewhere so that I'm able to set the right code to get it working without the hassle of trial-and-error.

### Acceptance-Criteria:

1. The value of the current axis that is available within the RPN scripts as "@" is shown somewhere on the page where I'm able to edit the RPN assignment.

### Background

For instance if I want to Assign the ADIRS knob function 
![image](https://github.com/MobiFlight/MobiFlight-Connector/assets/12555687/4e6bb2f6-7e5b-40a5-8980-df706fe4ce6c)
to an analogue input (simple poti) I can use something like 

```
@ 30000 <= if{ 0 (>L:A32NX_OVHD_ADIRS_IR_1_MODE_SELECTOR_KNOB) } 
30000 40000 @ rng if{ 1 (>L:A32NX_OVHD_ADIRS_IR_1_MODE_SELECTOR_KNOB) }
@ 40000 >= if{ 2 (>L:A32NX_OVHD_ADIRS_IR_1_MODE_SELECTOR_KNOB) } 
```

Unfortunately I need to guess the values (here 3000 and 4000) instead of being able just to turn the potentiometer to the right place and then "read" the value I need to put into the RPN to compare.

Did I just miss the right place, where I can read the current @ walue or is that simply missing at the moment?

Best regards from Walddorfhäslach,

Joe

**Comment by DocMoebiuz** [comment]
As a workaround for now: So when you turn on Logging in Settings you should immediately see log messages for the potentiometer that you are turning. You can move the dialog to a side so that it doesn't overlap the log window.

<img width="761" alt="image" src="https://github.com/MobiFlight/MobiFlight-Connector/assets/86157512/25043718-9348-4101-b31b-21c14bdc1392">

**Comment by joeherwig** [comment]
Hi Sebastian, Thanks for the fast response.
As I have the issue of jittering on two axis, which I can't filter out from the logging, I get a lot of clutter which makes it impossible to read it. 
So finally I found the logfile, but it's a mess as I get hundreds of entries with 17 analog inputs installed.
Is there any way to filter only "active" inputs?
Or to de-jitter (define something like "only report change when changed by minimum x)?

**Comment by JaimeLeon2** [comment]
The Mobiflight firmware has a built-in smoothing function that is already averaging the last 8 readings.  Additionally, the analog device configuration has a sensitivity setting that does what you described.
I suspect that you have an electrical noise problem in your panel and should take measures to filter it out.  7 segment displays are notorious for affecting potentiometer readings.  Capacitors might help.

**Comment by DocMoebiuz** [comment]
> Hi Sebastian, Thanks for the fast response. As I have the issue of jittering on two axis, which I can't filter out from the logging, I get a lot of clutter which makes it impossible to read it. So finally I found the logfile, but it's a mess as I get hundreds of entries with 17 analog inputs installed. Is there any way to filter only "active" inputs? Or to de-jitter (define something like "only report change when changed by minimum x)?

@joeherwig the axis are driven by potis that are connected via MobiFlight boards, not via a joystick? Is that correct. If that's the case I wonder if you have tried what @JaimeLeon2 suggested. There is a `sensitivity` option for the potentiometer in the board settings.

**Comment by joeherwig** [comment]
No, it's just Joysticks where I want to manage the assignments within Mobiflight instead of MSFS.

**Comment by DocMoebiuz** [comment]
The new UI already shows this value in the Raw value column.
The new input config wizard now shows the raw value and final value also inside the dialog as long as MobiFlight is running. We will add soon that you can also see the value update without MobIFlight being in "run" mode.

#3092
